androidApp/src/main/java/com/mbta/tid/mbta_app/android/map/HomeMapView.kt

```diff
@@ -282,17 +292,108 @@ fun HomeMapView(
 
             val context = LocalContext.current
 
+            val locationProvider = remember {
+                object : LocationProvider {
```

Comment:
Could we move this to it's own file? `HomeMapView` is pretty big already, I think it would help readability to pull this out. Nonblocking!

Reply:
Moved.